appveyor.yml

```diff
@@ -1,7 +1,7 @@
 # https://www.appveyor.com/docs/appveyor-yml/
 
 version: "{build}"
-
+image: Visual Studio 2022
```

Comment:
New image is required to use Ruby 2.7: [docs](https://www.appveyor.com/docs/windows-images-software/#ruby).